transactions/models.py

```diff
@@ -0,0 +1,59 @@
+from django.db import models
+
+# Create your models here.
+from blocks.models import Block
+
+
+class Transaction(models.Model):
+    '''
+    This represents a transaction(exchange of coins) on the blockchain
+    '''
+    hash = models.CharField(max_length=100, blank=True, null=True)
+    block = models.ForeignKey(Block, blank=True, null=True, on_delete=models.CASCADE, related_name='transactions')
```

Comment:
I am a little skeptical about Block deletion. If a block is deleted, all data for a transaction be deleted? Shouldn't we employ the strategy that if a block is lost, we cannot verify the relevant transactions and keep the actual transaction data within the chain? Correct me if I am wrong here.

Reply:
We may have to delete some blocks in case of a "chain reorganisation" i.e if someone broadcasts a different chain longer than ours, we will have to delete blocks that are not part of the longer chain....hence there needs to be a mechanism for deleting that block and the transactions inside.